scripts/reset.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Thanks for adding the script. I think it'd be good to add a comment at the top to clarify that this script should be run in the `prime-reportstream` repo (inside the `prime-router` folder)